quint/src/cliCommands.ts

```diff
@@ -312,6 +315,18 @@ export async function runTests(prev: TypecheckedStage): Promise<CLIProcedure<Tes
       maxSamples: testing.args.maxSamples,
       rng,
       verbosity: verbosityLevel,
+      onTrace: (index: number, name: string, status: string, vars: string[], states: QuintEx[]) => {
+        if (outputTemplate && outputTemplate.endsWith('.itf.json')) {
+          const filename = outputTemplate.replaceAll('{}', name).replaceAll('{#}', index)
```

Comment:
So if there's no `{#}`, the file will be overwritten with the latest run.
What if there's no `{}` and multiple tests?

Reply:
The arguments validator is checking for that. Initially, the check was in `runTests`, but I've decided to move it in the args parser.